Frida/Script.swift

```diff
@@ -214,4 +251,55 @@ public class Script: NSObject, NSCopying {
     private let releaseConnection: GClosureNotify = { data, _ in
         Unmanaged<SignalConnection<Script>>.fromOpaque(data!).release()
     }
+    
+    // MARK: - RPC Types
+    
+    @dynamicMemberLookup
+    public struct Exports {
+        unowned var script: Script?
```

Comment:
```suggestion
        unowned let script: Script
```

Or am I missing something? (My Swift is a little rusty.)

Reply:
Yeah you're right!